minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterControl.java

```diff
@@ -278,6 +279,13 @@ public synchronized void stop(ServerType server, String hostname) throws IOExcep
         if (managerProcess != null) {
           try {
             cluster.stopProcessWithTimeout(managerProcess, 30, TimeUnit.SECONDS);
+            try {
+              new ZooZap().zap(cluster.getServerContext().getSiteConfiguration(),
+                  new String[] {"-manager"});
```

Comment:
This would probably look nicer if ZooZap took varargs instead of a `String[]`

Should this go in the finally block instead of in the same block that's handling the 30 second timeout?